### PR TITLE
Allows pybroker to fetch adjusted stock market data from alpaca

### PR DESCRIPTION
**Pull request by MythicalDJ**
This should fix my issue in issue no #148 and allows pybroker to fetch adjusted market data from the alpaca API

### CODE REVIEW

**Comment by edtechre** [comment]
Thank you @MythicalDJ! Can you commit this to the dev branch instead?

**Comment by MythicalDJ** [comment]
Sure thing!

File: src/pybroker/strategy.py [hunk 1]
```diff
@@ -830,11 +830,13 @@ def __init__(
         start_date: Union[str, datetime],
         end_date: Union[str, datetime],
         config: Optional[StrategyConfig] = None,
+        adjust: Optional[any] = None,
     ):
         self._verify_data_source(data_source)
         self._data_source = data_source
         self._start_date = to_datetime(start_date)
         self._end_date = to_datetime(end_date)
+        self._adjust = adjust
```
Comment (edtechre) [inline src/pybroker/strategy.py:839]: No need to save to a field.

File: src/pybroker/config.py [hunk 2]
```diff
@@ -98,3 +98,4 @@ class StrategyConfig:
     return_signals: bool = field(default=False)
     return_stops: bool = field(default=False)
     round_test_result: bool = field(default=True)
+    adjust: Optional[any] = field(default=None)
```
Comment (edtechre) [inline src/pybroker/config.py:101]: Not needed.

File: src/pybroker/strategy.py [hunk 3]
```diff
@@ -830,11 +830,13 @@ def __init__(
         start_date: Union[str, datetime],
         end_date: Union[str, datetime],
         config: Optional[StrategyConfig] = None,
+        adjust: Optional[any] = None,
```
Comment (edtechre) [inline src/pybroker/strategy.py:833]: I believe this needs to be Any, capitalized.

**Review by edtechre — COMMENTED** [review]
Thank you!

**Comment by edtechre** [comment]
Added in https://github.com/edtechre/pybroker/commit/27422dee242d129b4e59a03b9c3812d34a51e741

**Pull request closed without merging**